packages/circuit-ui/components/NotificationInline/NotificationInline.docs.mdx

```diff
@@ -0,0 +1,35 @@
+import { Status, Props, Story } from '../../../../.storybook/components';
+import { NotificationToast } from '@sumup/circuit-ui';
+
+# NotificationInline
+
+<Status.Stable />
+
+<Story id="notification-notificationinline--base" />
+<Props />
+
+The Notification Inline component non-disruptively provides quick and contextual inline notification and it is integrated within the content section.
+Inline notifications are frequently used to provide additional guidance to the user, like positive feedback, confirmation about an action or errors in forms, cards or dialog boxes.
+
+## Component variations
+
+### Notification Inline variants
+
+<Story id="notification-notificationinline--variants" />
+
+- Use the 'Info' variant for informational, neutral messages.
+- Use the 'Confirm' variant the successful messages.
+- Use the 'Notify' variant for warning messages, and other information a user should be aware of.
+- Use the 'Alert' variant for error messages. Be descriptive and give users clear next steps.
+
+### Inline with headline and action button
+
+<Story id="notification-notificationinline--dismissable" />
+
+## Accessibility guidelines
+
+- Be concise, specific and short in the message.
+- Inline notifications are part of the content and should be placed near their related part of the content.
+- Do not cover other content with inline notifications.
+- In case you need to provide an action button, use NotificationInline component. Toast messages should not contain any interactive elements. Screen readers will announce raw text within the toast as it appears, without structure and without moving focus to the toast element.
+- Inline notifications should not be added to the page dynamically. For this use case, use Toast notifications, since only dynamic alerts include aria live attributes.
```

Comment:
Hmmmm this is complicated, because for example if we need to show server-side form submission errors, it's actually better to avoid the toast and use the inline (since the toast violates 2.2.1 timing adjustable)

Ideally developers who need to write a dynamic form submission notification use the `NotificationInline` but implement the live region themselves.

In the future we'll want to package this into form patterns, but in the meantime I'm not sure what the best approach is here, for documentation purposes. I'm just worried that right now we seem to be pointing from the `NotificationInline` to the toasts (because no live regions) and from the toasts back here (because no timing adjustable)—which could be quite confusing